src/main/java/frc/robot/RobotContainer.java

```diff
@@ -36,7 +34,7 @@ public class RobotContainer {
   public final PhotonCamera frontRightCamera = new PhotonCamera("front-right");
   public final PhotonCamera backLeftCamera = new PhotonCamera("back_left");
   public final PhotonCamera backRightCamera = new PhotonCamera("back_right");
-  public final PhotonCamera frontCenterCamera = new PhotonCamera("front_center");
+  public final PhotonCamera frontCenterCamera = new PhotonCamera("front-center");
```

Comment:
ℹ️ These are named inconsistently with some underscores and others hyphens -- is that alright?